scalableminds/webknossos
# Fix URL hash change

As a result of this fix, hash changes in the URL will now be reflected in the tracing without a reload :) (There's a slight delay, where the hash changes back to the original value first and then to the correct value - this is because the hash is always rebuilt using the store as a ground truth)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Check whether comment links still work
  - Create a comment with the string `#1` and click on it, the node with id 1 should be activated
- Change the URL hash
  - Change the position in the URL hash, the tracing position should change accordingly

### Issues:
- fixes #3707 

------
